Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -189,6 +200,33 @@ extension CommentRepliesViewController: CommentComposerViewDelegate {
   }
 }
 
+// MARK: - CommentCellDelegate
+
+extension CommentRepliesViewController: CommentCellDelegate {
+  func commentCellDidTapHeader(author _: Comment.Author) {
+    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
+
+    if #available(iOS 15.0, *) {
+      if let sheet = self.view.filter({ $0.tag == actionSheetTag }) {
+        sheet.removeFromSuperview()
+      }
+
+      let actionSheet =
+        UIHostingController(rootView: BlockUserActionSheetView(
+          blockUser: { self.blockUser() }
+      ))
+      actionSheet.view.tag = actionSheetTag
```

Comment:
The action sheet is only being added as a subview as needed. I'm giving it a tag so that we can avoid adding a new instance on every tap. See `line: 210`